src/main/scala/com/activegrid/models/LogConfigUpdater.scala

```diff
@@ -0,0 +1,36 @@
+package com.activegrid.models
+
+/**
+  * Created by sivag on 7/10/16.
+  */
+
+
+import org.apache.log4j._
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class LogConfigUpdater(implicit ex: ExecutionContext) {
+
+  val ROOT: String = "root"
+
+  def getLogger(logger: String): Logger = {
+    logger match {
+      case ROOT => getLogger(ROOT)
+      case _ => getLogger(logger)
+    }
+  }
+
+  def getLogLevel2(logger: String): Future[String] = Future {
```

Comment:
We have end points that return back  LOGGER related information. We couldn't find any use case but might required by other services. Now corrected the method definition.